lib/glific/state/state.ex

```diff
@@ -96,8 +96,25 @@ defmodule Glific.State do
   end
 
   @doc false
-  def get_flow(user, flow_id, is_forced) do
-    GenServer.call(__MODULE__, {:get_flow, %{user: user, flow_id: flow_id, is_forced: is_forced}})
+  def get_flow(user, flow_id, is_forced, retries \\ 0) do
+    GenServer.call(
+      __MODULE__,
+      {:get_flow, %{user: user, flow_id: flow_id, is_forced: is_forced}}
+    )
+  catch
+    :exit, {:timeout, _} = _reason ->
+      if retries < 1 do
+        log_msgq()
+        get_flow(user, flow_id, is_forced, retries + 1)
+      else
+        {:ok,
```

Comment:
should we log this here also, so we know how many succeeded after one retry, and how many still failed

Reply:
Will do. But retrying a GenServer.call still put this new message in the queue until the first one completes. So if first message took 8s to complete, if we retry right after the timeout, we basically only tries for 2s in the retry, since 3s we were waiting in the queue. So should we dynamically increase the timeout to 10s when we retry?. If our goal is to log the details then we might not need this. But if our goal is to check the retry success rate too, then maybe we might need to give ample time for the retry to perform.
@dlobo @AkhileshNegi